src/ascii/port.rs

```diff
@@ -14,9 +15,14 @@ use crate::{
         Alert, AnyResponse, Command, CommandWriter, Info, MaxPacketSize, Reply, Response,
         ResponseBuilder, Status, Target,
     },
+    convert::From as ThisFrom,
```

Comment:
No need for an alias. It shows up in the docs, which reads oddly. Instead just use the full path in the trait bounds.

Reply:
Removed during rebase